templates/workspace_services/guacamole/user_resources/guacamole-azure-import-reviewvm/apply_patch.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Version of TRE to use as reference
+AZURE_TRE_VERSION="0.5.0"
+
+# Download the repository of the AZURE_TRE_VERSION locally
+# and unpack the templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm into local directory
+# --strip-components=6 removes the leading components of path, leaving only what's inside guacamole-azure-windowsvm
+# --skip-old-files keeps the files we already have in current directory
+curl -o azuretre.tar.gz -L "https://github.com/microsoft/AzureTRE/archive/refs/tags/v${AZURE_TRE_VERSION}.tar.gz" \
+  && tar -xzf azuretre.tar.gz "AzureTRE-${AZURE_TRE_VERSION}/templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm"  --strip-components=6 --skip-old-files \
```

Comment:
why is this '--strip-components=6` when the other file is `--strip-components=5`?
(genuine question....both may be correct)

Reply:
It's because --strip-components=6 extracts all of the contents of guacamole-azure-windowsvm into current directory, without the guacamole-azure-windowsvm directory itself. Whereas --strip-components=5 keeps the guacamole-azure-windowsvm directory (because I want it to be separate)

Reply:
makes sense